pkgs/by-name/xd/xdg-desktop-portal-cosmic/package.nix

```diff
@@ -7,33 +7,44 @@
 , libxkbcommon
 , pipewire
 , wayland
+, gst_all_1
 }:
 
-rustPlatform.buildRustPackage rec {
+rustPlatform.buildRustPackage {
   pname = "xdg-desktop-portal-cosmic";
-  version = "unstable-2023-12-07";
+  version = "0-unstable-2024-02-23";
 
   src = fetchFromGitHub {
     owner = "pop-os";
-    repo = pname;
-    rev = "23b3e5a1b9fa76e30266f29949d54e97c2fadf6e";
-    hash = "sha256-AqwJ3bV8Xz0MpY/ZmWgE9vNJIACX5SVeIYbSewyG/Bs=";
+    repo = "xdg-desktop-portal-cosmic";
+    rev = "f6d394e5d05b6acdf7d39d5211d4dca02b419e39";
```

Comment:
Last commit is `5f38044f15656fb946c146b03f479d0931a093e0` from ` May 30, 2024`

Care to update? I can review it once you have updated it.